client/src/components/SiteSearch/SiteSearchListing/SiteSearchListing.js

```diff
@@ -109,22 +109,25 @@ const SiteSearchListing = ({ results, loading, error, filter }) => {
           <LoadingSpinner />
         </div>
       )}
-      {hasNoResults && (
+
+      {!loading && hasNoResults && (
```

Comment:
Since there is only one component rendered when `loading` is `true`, we could pull out the `loading` logic and `LoadingSpinner` component rendering to a second `return` block, above. e.g.
```
...
  const lastPageIndex = Math.ceil(results.count / 10);
  if (loading) {
    return (
      <div styleName={containerStyleNames}>
        <div styleName="placeholder">
          <LoadingSpinner />
        </div>
      </div>
    );
  }
  return (
    <div styleName={containerStyleNames}>
      <SiteSearchSearchbar />
      <h5 styleName="header">{FILTER_MAPPING[filter]}</h5>
      ...
```

Alternatively, we could wrap everything after line 113 in a `{!loading && <>...` block